src/content/learn/passing-props-to-a-component.md

```diff
@@ -249,29 +249,29 @@ function Profile(props) {
 }
 ```
 
-This forwards all of `Profile`'s props to the `Avatar` without listing each of their names.
+Isso encaminha todas as props de `Profile` ao `Avatar` sem listar cada um de seus nomes.
 
-**Use spread syntax with restraint.** If you're using it in every other component, something is wrong. Often, it indicates that you should split your components and pass children as JSX. More on that next!
+**Use a sintaxe de espalhamento com cuidado.** Se você está a utilizando em quase todos os componentes, algo está errado. Muitas vezes, isso indica que você deveria dividir seus componentes e passar filhos como JSX. Mais sobre isso a seguir!
 
-## Passing JSX as children {/*passing-jsx-as-children*/}
+## Passando JSX como `children` {/*passing-jsx-as-children*/}
 
-It is common to nest built-in browser tags:
+É comum aninhar tags embutidas no navegador:
 
 ```js
 <div>
   <img />
 </div>
 ```
 
-Sometimes you'll want to nest your own components the same way:
+Às vezes você desejará aninhar seus próprios componentes da mesma forma:
 
 ```js
 <Card>
   <Avatar />
 </Card>
 ```
 
-When you nest content inside a JSX tag, the parent component will receive that content in a prop called `children`. For example, the `Card` component below will receive a `children` prop set to `<Avatar />` and render it in a wrapper div:
+Quando você aninha conteúdo dentro de uma tag JSX, o componente pai irá recebê esse conteúdo em uma prop chamada `children`. Por exemplo, o componente `Card` abaixo receberá a prop `children` definida como `<Avatar />` e o renderizará em uma div a qual o encapsula:
```

Comment:
Olá @TarsiSurdi, apenas mais um pequeno detalhe aqui. De acordo com o [glossário](https://github.com/reactjs/pt-br.react.dev/blob/main/GLOSSARY.md) não deve-se traduzir a palavra `wrapper`.

```suggestion
Quando você aninha conteúdo dentro de uma tag JSX, o componente pai irá receber esse conteúdo em uma prop chamada `children`. Por exemplo, o componente `Card` abaixo receberá a prop `children` definida como `<Avatar />` e o renderizará em uma wrapper div:
```

Reply:
Obrigado! Me confundi entre "to wrap" (que deve ser traduzido) e "wrapper" :(